src/OperatingSystemApp/wobblyLockscreen.js

```diff
@@ -0,0 +1,51 @@
+/* exported WobblyLockscreen */
+
+const {GLib, GObject} = imports.gi;
+
+const {Lockscreen} = imports.lockscreen;
+
+var WobblyLockscreen = GObject.registerClass({
+    GTypeName: 'WobblyLockscreen',
+    CssName: 'wobblylockscreen',
+}, class WobblyLockscreen extends Lockscreen {
+    _init(props = {}) {
+        super._init(props);
+        this._timeout = null;
+        this.connect('destroy', () => {
+            if (this._timeout) {
+                this._manager.setLockTried(this.lock, false);
+                GLib.Source.remove(this._timeout);
+            }
+        });
+    }
+
+    _updateBackground() {
+        super._updateBackground();
+        const assetsPath = this.getAssetsPath();
+        if (this._manager.getTrapSequence(this.lock) === 0)
+            this._playbin.background = `file://${assetsPath}/trap0`;
```

Comment:
This is a pre-existing situation, but do you know why we didn't include the lockscreen assets in the GResource and instead are installing them to `pkgdatadir`?